omas/tests/test_omas_utils.py

```diff
@@ -64,10 +65,12 @@ def test_different_ods(self):
         diff_prof5 = different_ods(ods2, ods3)
         assert isinstance(diff_prof5, list)
         assert 'name' in ' '.join(diff_prof5)
+        return
```

Comment:
@eldond is there a reason for adding `return` at the end of each function. That's not necessary, since the function will also return `None`

Reply:
It's easier to read and easier to fold

Reply:
To clarify, I've found that some of these classes with a ton of methods get messy without nice clear breaks between methods. So, I like to put in the empty returns. It's just a preference/style thing. Go ahead and delete them if you don't like them.

Reply:
Yeah, I do not like them, and I would not want to add it to every function/method to be consistent. For now I can survive with these empty return statements and perhaps I'll remove them later.

I am not sure I understand what you mean by `easier to fold` in PyCharm. You'll have to show me!

Reply:
Sometimes the last line of the function is also the last line in an indented block for if, else, for, or something. Then it seems to default to folding the if/else/for block instead of the whole function. The same thing for classes, where the last line of the class is the last line of a method, unless a terminating comment or something is added.